scripts/coala_quickstart

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+import sys
+
+def main():
+  files_glob = input("Please enter the files glob: ")
+  coafile = open(".coafile", "a+")
```

Comment:
why is this no .py file? For some platforms this is needed, that might be your issue actually